js_modules/dagit/packages/core/src/asset-graph/AssetGraphExplorer.tsx

```diff
@@ -356,9 +368,12 @@ export const AssetGraphExplorerWithData: React.FC<
                         }}
                         style={{overflow: 'visible'}}
                       >
-                        {!graphNode || !graphNode.definition.opNames.length ? (
-                          <ForeignNode assetKey={{path}} />
-                        ) : scale < MINIMAL_SCALE ? (
+                        {false && (!graphNode || !graphNode.definition.opNames.length) ? (
```

Comment:
I think this isn't quite right but it's fixed in the PR I stacked on this one,  let's leave it for now